react/.storybook/preview-head.html

```diff
@@ -0,0 +1,25 @@
+<script>
+
+const runOnPageChange = () => {
```

Comment:
I left comments on the same code here on https://github.com/sparkdesignsystem/spark-design-system/pull/2602

Reply:
@RCopeland It actually doesn't work correctly if I wrap it.